### PR TITLE
Fix remaining broken links

### PR DESCRIPTION
**Pull request by noon-dawg**
Fixed remaining broken links.

### CODE REVIEW

File: openapi/description.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Rebilly also offers [FramePay](https://docs.rebilly.com/docs/developer-docs/fram
  a client-side iFrame-based solution to help
 create payment tokens while minimizing PCI DSS compliance burdens
 and maximizing the customization ability. [FramePay](https://docs.rebilly.com/docs/developer-docs/framepay/)
-is interacting with the [payment tokens creation operation](#operation/PostToken).
+is interacting with the [payment tokens creation operation](https://www.rebilly.com/docs/billing/api/tag/Payment-Tokens/#tag/Payment-Tokens/operation/PostToken).
```
Comment (ashkarpetin) [inline openapi/description.md:63]: it seems strange if I open https://api-reference.rebilly.com/#section/SDKs and click on the `payment tokens creation operation` link and got redirected to a different website.

Comment (noon-dawg) [reply openapi/description.md]: Thanks for point this out! I've updated the link in my last commit ([47b1af5](https://github.com/Rebilly/api-definitions/pull/749/commits/47b1af5f4a1be1600029589a59b18983e2086ca4)).

The images aren't working in the live or preview environments. I'm having trouble figuring this out

File: openapi/description.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Rebilly offers four forms of authentication: secret key, publishable key, JSON W
 
 To create or manage API keys, select one of the following:
 
-- Use the Rebilly UI: In the left navigation bar, click ![automation icon](/images/automation.svg), **Integrations**, **Custom integrations**, then click **API keys**.
```
Comment (ashkarpetin) [inline openapi/description.md:31]: Should we remove `automation.svg` file?

**Comment by ashkarpetin** [comment]
Please remove `web/.DS_Store` file

File: openapi/description.md (hunk 2 above)
Comment (noon-dawg) [reply openapi/description.md:31]: removed

**Review by ashkarpetin — APPROVED** [review]

**Review by XaosSintez — APPROVED** [review]

**Pull request merged**